### ISSUE TITLE
Completely misaligned 

### ISSUE THREAD

**Issue by mradzinski** [Yalantis/JellyToolbar#6]
Following the demo (Literally copying and pasting but just changing some colors) the Toolbar presents a total misalignation. Tested in API 26, 25, 24, 23, 22, 21 and 19 and not working in any of them.

<img width="400" alt="captura de pantalla 2017-07-03 a la s 18 19 43" src="https://user-images.githubusercontent.com/4214209/27808139-3baf201c-601c-11e7-8ea0-94cb83a0284f.png">

**Comment by hoelee** [comment]
Did you solve that?

**Comment by mradzinski** [comment]
Yeah, it was actually project related. Are you extending the toolbar to cover the status bars?

**Comment by hoelee** [comment]
At the end didn't use this, found another problem - the logo will cover OptionsMenu... I can't figure out how to fix that...
